book/src/libloading.md

```diff
@@ -0,0 +1,8 @@
+It is also possible to generate bindings for loading a shared library via the `libloading` crate.
```

Comment:
```suggestion
It is also possible to generate bindings for dynamically loading a shared library via the [`libloading`](https://docs.rs/libloading/latest/libloading/) crate.
```

I think it would be helpful to link the documentation of libloading, and to emphasize the **dynamic** / on-demand part. Bindgen will also work perfectly fine with shared libraries, if you tell rust to link against that library. Libloading allows us to lazily load libraries, instead of during startup.

Reply:
Thanks for your response. That is true of course, thank you for your suggestions. :-)